crypto/evp/p_lib.c

```diff
@@ -2184,13 +2184,10 @@ const OSSL_PARAM *EVP_PKEY_settable_params(const EVP_PKEY *pkey)
 
 int EVP_PKEY_set_params(EVP_PKEY *pkey, OSSL_PARAM params[])
 {
-    int ret = pkey != NULL
+    pkey->dirty_cnt++;
+    return pkey != NULL
```

Comment:
Now it's dereferencing pkey before checking it for NULL :(

Reply:
Oops

Reply:
Fixed